layouts/topnav/index.tsx

```diff
@@ -0,0 +1,52 @@
+import Image from 'next/image'
+
+import Styled from './styled'
+
+const routes = [
+  'token',
+  'litepaper',
+  'builder',
+  'marketplace',
+  'about',
+  'faq'
```

Comment:
This accounts-frontend project will be like a "microservice frontend", these Nav Links should go to: www.app.tryspace.com
For example: "builder" NavLink should redirect to: https://app.tryspace.com/create-space
faq: https://www.tryspace.com/faq 
etc...